src/pages/academy/blog/preview/index.js

```diff
@@ -0,0 +1,608 @@
+    }
+    & a {
+        font-size: 16px;
+        color: var(--color-red);
+        text-decoration: none;
+        cursor: pointer;
+
+        &:hover {
+            text-decoration: underline;
+        }
+    }
+    & img {
+        max-width: 100%;
+        height: auto;
+        display: block;
+        margin: auto;
+        margin-top: 16px;
+    }
+    & img[width='full'] {
+        margin-left: calc(50% - 50vw);
+        margin-right: calc(50% - 50vw);
+        max-width: 100vw;
+    }
+    & h1 {
+        font-size: 64px;
+        line-height: 80px;
+        margin-top: 32px;
+        font-weight: bold;
+    }
+    & h2 {
+        font-size: 48px;
+        line-height: 60px;
+        margin-top: 32px;
+        font-weight: bold;
+    }
+    & h3 {
+        font-size: 32px;
+        line-height: 40px;
+        margin-top: 40px;
+        font-weight: bold;
+
+        & + p {
+            margin-top: 8px;
+        }
+    }
+    & h4 {
+        font-size: 24px;
+        line-height: 36px;
+        margin-top: 40px;
+        font-weight: bold;
+
+        & + p {
+            margin-top: 8px;
+        }
+    }
+    & h5 {
+        font-size: 20px;
+        line-height: 30px;
+        margin-top: 24px;
+        font-weight: bold;
+    }
+    & h6 {
+        font-size: 16px;
+        line-height: 24px;
+        margin-top: 40px;
+        font-weight: bold;
+    }
+    @media ${device.laptop} {
+        max-width: none;
+
+        & p {
+            font-size: 14px;
+        }
+        & ul {
+            > li {
+                font-size: 14px;
+                line-height: 20px;
+            }
+        }
+        & h1 {
+            font-size: 32px;
+            line-height: 40px;
+        }
+        & h2 {
+            font-size: 28px;
+            line-height: 34px;
+        }
+        & h3 {
+            font-size: 24px;
+            line-height: 30px;
+
+            & + p {
+                margin-top: 8px;
+            }
+        }
+        & h4 {
+            font-size: 18px;
+            line-height: 26px;
+
+            & + p {
+                margin-top: 8px;
+            }
+        }
+        & h5 {
+            font-size: 16px;
+            line-height: 24px;
+        }
+        & h6 {
+            font-size: 14px;
+            line-height: 20px;
+        }
+    }
+`
+const SocialComponentsWrapper = styled(Flex)`
+    justify-content: space-between;
+    border-top: 1px solid var(--color-grey-6);
+    padding-top: 24px;
+    margin: 40px 0;
+`
+const LeftSocialComponents = styled.div`
+    width: 10px;
+`
+const RightSocialComponents = styled.div`
+    width: auto;
+`
+
+const convertDate = (date) => {
+    const newdate = new Date(date)
+    return (
+        newdate.toLocaleString('en', { day: 'numeric' }) +
+        ' ' +
+        newdate.toLocaleString('en', { month: 'long' }) +
+        ' ' +
+        newdate.toLocaleString('en', { year: 'numeric' })
+    )
+}
+
+export const query_preview = graphql`
```

Comment:
```suggestion
const query_preview = graphql`
```

Reply:
We can remove this if it's a static query.

Reply:
amended in latest commit